<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qemuLaunchWizard</class>
 <widget class="QWizard" name="qemuLaunchWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QEMU Configuration Wizard</string>
  </property>
  <widget class="QWizardPage" name="qemuLaunchWizardNamePage">
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="descritionLabel">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>100</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="descriptionPlainTextEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deletePushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="qemuLaunchWizardImagePage">
   <widget class="QFrame" name="frame_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>361</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="imageSelectButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../resources/document-open.png</normaloff>
       <normalon>../resources/document-open.png</normalon>../resources/document-open.png</iconset>
     </property>
    </widget>
    <widget class="QLineEdit" name="imageLineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="imageLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Image File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modImagePushButton">
     <property name="geometry">
      <rect>
       <x>178</x>
       <y>50</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Modify Image Contents</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>361</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="configLineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="configSelectButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../resources/document-open.png</normaloff>
       <normalon>../resources/document-open.png</normalon>../resources/document-open.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="configLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Config File</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWizardPage" name="qemuLaunchWizardNetworkPage">
   <property name="title">
    <string/>
   </property>
   <widget class="QFrame" name="frame_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>361</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="ipLineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>000.000.000.000</string>
     </property>
    </widget>
    <widget class="QLabel" name="gatewayLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Gateway</string>
     </property>
    </widget>
    <widget class="QLabel" name="ipLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>IP Address</string>
     </property>
    </widget>
    <widget class="QLabel" name="subnetMaskLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Subnet Mask</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="gatewayLineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>000.000.000.000</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="subnetMaskLineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>000.000.000.000</string>
     </property>
    </widget>
    <widget class="QLabel" name="ifaceLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Interface Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ifaceLineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWizardPage" name="qemuLaunchWizardDevicePage">
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="deviceTypeLabel">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Device Type</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="deviceLabel">
     <property name="geometry">
      <rect>
       <x>38</x>
       <y>50</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Device</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="deviceTypeComboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="deviceComboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addDevicePushButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>371</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListView" name="deviceListView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>291</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="removeDevicePushButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>80</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editDevicePushButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
